octavia_f5/network/drivers/neutron/hierarchical_port_binding.py

```diff
@@ -63,8 +51,9 @@ def allocate_vip(self, load_balancer):
                          'network_id': load_balancer.vip.network_id,
                          'admin_state_up': False,
                          'device_id': 'lb-{0}'.format(load_balancer.id),
-                         'device_owner': LBAASV2_OWNER,
-                         'binding:host_id': 'f512-03',
+                         'device_owner': constants.DEVICE_OWNER_LISTENER,
+                         # TODO: needs to be auto-scheduled by agent
+                         'binding:host_id': 'neutron-f5-f512-01',
```

Comment:
Would it maybe be better to export 'neutron-f5-f512-01' into a constant or even configuration option?
Because I suppose this host ID is only for testing purposes (?)

Reply:
Correct, this should be fixed by Issue #4, so this is currently just for our testing purpose